src/category.py

```diff
@@ -10,13 +10,26 @@ class Category:
     # Свойства класса.
     name: str
     description: str
-    products: list
+    __products: list
 
     def __init__(self, name, description, products):
         # Инициализация класса, передача значений атрибутов класса.
         self.name = name
         self.description = description
-        self.products = products
+        self.__products = products
 
         Category.category_count += 1
         Category.product_count += len(products)
+
+    # Геттер для приватного атрибута класса
+    @property
+    def products(self):
+        return "\n".join(
+            f"{product.name}, " f"{product.price} руб. Остаток: " f"{product.quantity} шт."
```

Comment:
Слишком много f-строк
```suggestion
            f"{product.name}, {product.price} руб. Остаток: {product.quantity} шт."
```

Reply:
Здесь перенос был, так как линтер ругался. А другой линтер все в одну строку кинул. Сам не понял как так вышло.